learn/how-tos/Integrating-JUnit-tests-for-WaveMaker-application.md

```diff
@@ -0,0 +1,128 @@
+# Integrating JUnit tests for WaveMaker application
+
+
+### OVERVIEW
+
+JUnit is a unit testing framework for the Java programming language. JUnit has been important in the development of test-driven development. These tests are required whether imported services are working as expected or not, before proceeding to run/deployment of the application. This document demonstrates how to integrate JUnit tests in an application developed by WaveMaker RAD.
+
+### GOALS
+
+1. Integrate JUnit tests in the WaveMaker application
+2. Writing sample test cases for services.
+
+### PREREQUISITES 
+
+1. IDE (IntelliJ IDEA, Eclipse)
+2. WaveMaker Application
+3. Git
+4. Maven
+
+### CONFIGURATION
+
+1. Export the WaveMaker application as a zip file.
+2. Import this zip file into IDE (Use [WaveMaker workspace sync plugin](/learn/how-tos/synchronizing-wavemaker-apps-ides-beta)).
+3. Add the following dependencies in pom.xml file.
```

Comment:
Please mention under which section they need to be added like under <dependencies> etc.

Reply:
Added the section where the dependencies are to be added "under dependencies tag".